start_cerberus.py

```diff
@@ -138,8 +139,18 @@ def main(cfg):
             logging.info("Detailed inspection of failed components has been enabled")
             inspect.delete_inspect_directory()
 
-        # get list of all master nodes to verify scheduling
-        master_nodes = kubecli.list_nodes("node-role.kubernetes.io/master")
+        # get list of all master nodes with provided labels in the config
+        master_label = watch_master_schedulable["label"]
+        master_nodes = []
+        if watch_master_schedulable["enabled"]:
+            nodes = kubecli.list_nodes(master_label)
+            if len(nodes) == 0:
+                logging.error("No master node found for the label %s \
+                    Either set this label to False or add proper labels \
+                    in the config" % (master_label))
+                sys.exit(1)
+            else:
+                master_nodes.append(nodes)
```

Comment:
I noticed that the master_nodes variable here is actually a double array using a logging statement. 
`Process master nodes [['ip-10-0-133-137.ec2.internal', 'ip-10-0-151-124.ec2.internal', 'ip-10-0-163-207.ec2.internal']]`
This doesn't really matter for this but might be nice to be set as a single array for if we ever need to use this master nodes variable.

Reply:
nice catch, fixed :+1: